File: Problems/Find All Numbers Disappeared in an Array/workArea/findNum.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        duplicateFilter = []
        for i in nums:
            i = abs(i)
            
            if nums[i-1] > 0:
                nums[i-1] *= -1
            '''else:
                duplicateFilter.append(i+1)'''
                
        # print(duplicateFilter)
        # print(nums)
        for j in range(len(nums)):
            if(nums[j] > 0):
                duplicateFilter.append(j+1)
        # return duplicateFilter
        # for i in range(len())
        '''for j in range(len(duplicateFilter)):
            # print(j)
            duplicateFilter[j] = len(nums)-duplicateFilter[j];'''
        # print(duplicateFilter.revese())
        # duplicateFilter.reverse()
        
        return duplicateFilter
        
        

File: Problems/Find All Numbers Disappeared in an Array/workArea/findNum.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDisappearedNumbers = function(nums) {
    duplicateFilter = []
    for(let i of nums){
        i = Math.abs(i);
            
        if(nums[i-1] > 0) nums[i-1] *= -1;
    }
    // console.log(nums);
    for(let j=0; j <nums.length; j++){
        if(nums[j] > 0) {
            duplicateFilter.push(j+1);
        }
    }
    
    return duplicateFilter;
        // console.log(j)
};